### ISSUE TITLE
Uncaught TypeError: editorElement.getModel is not a function

### ISSUE THREAD

**Issue by akondrasev** [facebook/nuclide#1086]
[Enter steps to reproduce:]

1. Open file
2. Right click -> Source control -> Revert

**Atom**: 1.15.0 ia32
**Electron**: 1.3.13
**OS**: Microsoft Windows 10 Pro
**Thrown From**: [nuclide](https://github.com/facebook/nuclide) package 0.214.0

### Stack Trace

Uncaught TypeError: editorElement.getModel is not a function

```
At C:\Users\ankon\.atom\packages\nuclide\pkg\nuclide-hg-repository\lib\main.js:111

TypeError: editorElement.getModel is not a function
  at /packages/nuclide/pkg/nuclide-hg-repository/lib/main.js:111:67)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:259:29)
  at CommandRegistry.handleCommandEvent (/app.asar/src/command-registry.js:3:59)
  at CommandRegistry.module.exports.CommandRegistry.dispatch (/app.asar/src/command-registry.js:160:19)
  at AtomEnvironment.module.exports.AtomEnvironment.dispatchContextMenuCommand (/app.asar/src/atom-environment.js:1168:28)
  at EventEmitter.outerCallback (/app.asar/src/application-delegate.js:325:25)
  at emitThree (events.js:116:13)
  at EventEmitter.emit (events.js:194:7)

```

### Commands

```
     -6:08.8.0 intentions:highlight (input.hidden-input)
  2x -6:08.6.0 core:save (input.hidden-input)
     -6:07.1.0 intentions:highlight (input.hidden-input)
     -6:06.9.0 core:save (input.hidden-input)
     -3:38.6.0 nuclide-blame:toggle-blame (span.syntax--meta.syntax--brace.syntax--round.syntax--js)
     -1:56.5.0 intentions:highlight (input.hidden-input)
  2x -1:56.4.0 core:copy (input.hidden-input)
     -0:35.1.0 intentions:highlight (input.hidden-input)
  8x -0:34.9.0 core:undo (input.hidden-input)
     -0:31.4.0 intentions:highlight (input.hidden-input)
     -0:31.3.0 core:save (input.hidden-input)
  2x -0:28.2.0 intentions:highlight (input.hidden-input)
  3x -0:27.6.0 core:undo (input.hidden-input)
     -0:26.2.0 intentions:highlight (input.hidden-input)
     -0:26.1.0 core:save (input.hidden-input)
     -0:17.5.0 nuclide-hg-repository:confirm-and-revert (div.gutter.nuclide-blame)
```

### Non-Core Packages

```
atom-beautify 0.29.17 
atom-hg 2.0.14 
atom-ternjs 0.18.2 
atom-typescript 11.0.1 
atom-webstorm-syntax-theme 0.0.2 
busy-signal 1.3.0 
copy-path 0.4.1 
ctrl-dir-scroll 0.2.3 
custom-syntax-theme 0.0.0 
file-icons 2.1.2 
haskell-grammar 0.4.0 
highlight-selected 0.13.1 
intentions 1.1.2 
language-babel 2.57.2 
language-graphql 0.8.0 
language-ini 1.19.0 
language-lua 0.9.11 
language-ocaml 1.1.2 
language-swift 0.5.0 
language-thrift 1.0.2 
linter 2.1.2 
linter-ui-default 1.2.2 
merge-conflicts 1.4.4 
minimap 4.27.0 
minimap-highlight-selected 4.6.0 
nuclide 0.214.0 
nuclide-format-js 0.0.36 
platformio-ide-terminal 2.5.0 
restore-windows 0.4.1 
set-syntax 0.3.2 
sort-lines 0.14.0 
tool-bar 1.1.0 
```

**Comment by rightparen** [comment]
Thanks for reporting.  I can repro this issue locally and will take a look.

**Comment by rightparen** [comment]
This should be fixed by 8dfd2944f5f84122120ae07e99707829cc2b3244, which will be included in this week's release.
